nautobot/ipam/tests/test_models.py

```diff
@@ -381,15 +381,27 @@ def test_duplicate_vrf_unique(self):
 
 
 class TestIPAddress(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.status = Status.objects.get_for_model(IPAddress).first()
+
     def test_create_and_get_ipaddress_using_host_and_prefix_length(self):
         """Assert bug https://github.com/nautobot/nautobot/issues/4733 Unable to retrieve IPAddress using `address`
         args if it was created using host and prefix_length is resolved"""
-        status = Status.objects.get_for_model(IPAddress).first()
-        host = "10.35.1.251"
-        prefix_length = 32
-        ipaddress = IPAddress.objects.create(host=host, prefix_length=prefix_length, status=status)
+        ipaddress = IPAddress.objects.create(host="10.35.1.251", prefix_length=32, status=self.status)
         self.assertTrue(IPAddress.objects.filter(address=str(ipaddress)).exists())
 
+    def test_varbinary_ip_fields_with_empty_values_do_not_violate_not_null_constrains(self):
+        # Assert that an error is triggered when the host is not provided.
+        # Initially, VarbinaryIPField fields with None values are stored as the binary representation of b'',
+        # thereby bypassing the Not Null Constraint check.
+        with self.assertRaises(IntegrityError) as err:
+            IPAddress.objects.create(prefix_length=32, status=self.status)
+        self.assertIn(
```

Comment:
Is this the same error message in postgres and mysql?

Reply:
```
 AssertionError: 'null value in column "host" of relation "ipam_ipaddress" violates not-null constraint' not found in '(1048, "Column \'host\' cannot be null")'
```